index.js

```diff
@@ -0,0 +1,29 @@
+function navItemsToggle() {
+  const humBurger = document.getElementsByClassName('menu-button');
+  const navItems = document.getElementsByClassName('nav-items');
+  navItems[0].classList.toggle('nav-toggle-show');
+  navItems[0].classList.remove('text-center');
+  navItems[0].classList.remove('align-items-center');
+  Object.entries(navItems[0].children).forEach((item) => {
+    item[1].classList.toggle('nav-item-show');
+    item[1].addEventListener('click', () => {
+      navItems[0].classList.remove('nav-toggle-show');
+      Object.entries(navItems[0].children).forEach((item, index) => {
+        item[1].classList.remove('nav-item-show');
+        if (index < 3) {
+          humBurger[0].children[index].classList.remove(`checked-${index + 1}`);
+        }
+      });
+    });
+  });
+}
+
+function humburgerToggle() {
+  const humBurger = document.getElementsByClassName('menu-button');
```

Comment:
- [ ] [REQUIRED] Kindly refer to my last attached image and use that to make sure that the hamburger icon has a reduced width so it can maintain the size of the one in the Figma design. Also, let's change it's color to the appropriate color the Figma has. This will provide your application with a better UI interface.